src/endpoints/tv-shows.ts

```diff
@@ -33,19 +34,35 @@ import {
 
 const BASE_TV = '/tv';
 
+export interface TvImageSearchOptions extends LanguageOption {
```

Comment:
Can we move this to the appropriate types file?

Reply:
OK, moved :)